server/graphql/v2/interface/Transaction.js

```diff
@@ -151,5 +167,31 @@ export const TransactionFields = () => {
         return transaction.createdAt;
       },
     },
+    expense: {
+      type: Expense,
+      resolve(transaction) {
+        if (transaction.ExpenseId) {
+          return transaction.getExpense({ paranoid: false });
+        } else {
+          return null;
+        }
+      },
+    },
+    order: {
+      type: Order,
+      resolve(transaction) {
+        if (transaction.OrderId) {
+          return transaction.getOrder({ paranoid: false });
```

Comment:
Let's use a loader to prevent N+1 queries

```suggestion
          return req.loaders.Order.byId.load(transaction.OrderId);
```